parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4825,7 +4825,11 @@ SqlNode ConditionalStmt() :
     final SqlNode e;
 }
 {
-    e = IfStmt()
+    (
+        e = IfStmt()
```

Comment:
nit: Order alphabetically.

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/263/commits/59ec4c1cd314b7c1b1161ac20c8f9361d1c3c2ec